test-integration/operator-tests/src/test/java/org/infinispan/operator/ClientAuthenticationIT.java

```diff
@@ -0,0 +1,96 @@
+package org.infinispan.operator;
+
+import cz.xtf.builder.builders.SecretBuilder;
+import cz.xtf.client.Http;
+import cz.xtf.core.http.Https;
+import cz.xtf.core.openshift.OpenShift;
+import cz.xtf.core.openshift.OpenShifts;
+import cz.xtf.junit5.annotations.CleanBeforeAll;
+import io.fabric8.kubernetes.api.model.Secret;
+import lombok.extern.slf4j.Slf4j;
+import org.assertj.core.api.Assertions;
+import org.infinispan.Infinispan;
+import org.infinispan.Infinispans;
+import org.infinispan.TestServer;
+import org.infinispan.identities.Credentials;
+import org.infinispan.util.CleanUpValidator;
+import org.infinispan.util.KeystoreGenerator;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Paths;
+
+/**
+ * ClientAuthenticationIT tests if clients are authenticated.
+ * Requires a trust store that contains all public client certificates in addition to the root CA certificate.
+ * Only clients that present a signed certificate can connect to Data Grid.
+ */
+@Slf4j
```

Comment:
This annotation provides logger within the class accessible via `log`. Eg.: `log.info("Info message")`. Since you are not using it, it can be removed